README.md

```diff
@@ -80,6 +80,39 @@ run `tensorboard --logdir=lightning_logs/` in a new window to track training pro
 ![image](https://user-images.githubusercontent.com/943003/203165744-68da33fa-967f-45a7-956e-f0fe0114f9cc.png)
 
 The training process will generate one plot at the end of every epoch, using a sample from the validation set
-- the first row is the spectrogram of the audio file
-- the second row is the actual map associated with the audio file in its signal representation
-- the third and fourth rows are signal representations of the maps produced by the model
+
+-   the first row is the spectrogram of the audio file
+-   the second row is the actual map associated with the audio file in its signal representation
+-   the third and fourth rows are signal representations of the maps produced by the model
+
+## 💻 Windows Batch Setup
+
+-   **Requirements**
+    -   🐍 Python 3.8 (via Microsoft Store, or python.org)
+
+#### Installation
+
+Install the source code directly through github, or with the git clone command:
+
+`git clone https://github.com/jaswon/osu-dreamer`
+
+#### Usage
+
+**INSTALL ALL REQUIREMENTS BY THIS POINT**
+
+Setup from this point is pretty simple, navigate into the osu-dreamer directory and then into the **shell_scripts** folder, this is where all the batch scripts are stored.
+
+First, you will need to run "! Install.bat", this will install osu-dreamer and all of its dependencies. Optionally you can install tensorboard and mathplotlib to view training statistics.
```

Comment:
```suggestion
First, you will need to run `! Install.bat`, this will install osu-dreamer and all of its dependencies. Optionally you can install tensorboard and mathplotlib to view training statistics.
```